enterprise/frontend/src/embedding-sdk/store/use-sdk-selector.ts

```diff
@@ -17,5 +21,6 @@ export const useSdkSelector: TypedUseSelectorHook<SdkStoreState> = (
     throw new Error(USE_OUTSIDE_OF_CONTEXT_MESSAGE);
   }
 
-  return useSelector(selector, options);
+  // @ts-expect-error -- weird error on the options type
```

Comment:
I couldn't find an easy solution, the resulting hook is typed correct as `TypedUseSelectorHook<SdkStoreState>` so the error shouldn't affect us